WolvenKit.App/Models/ModInfo.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using WolvenKit.Modkit.RED4.Sounds;
+
```

Comment:
I like to keep one class per file usually, so I would split these classes in their own files. But it is up to you guys how you like it.